awx/ui_next/src/App.jsx

```diff
@@ -39,43 +55,56 @@ function App() {
   const match = useRouteMatch();
   const { hash, search, pathname } = useLocation();
 
+  i18n.loadLocaleData('en', { plurals: en });
```

Comment:
I would recommend just loading the initial catalog (for example english) and then load the other catalogs dynamically. Check out https://lingui.js.org/ref/loader.html